### DISCUSSION TITLE
Add 'Stale' Label

### DISCUSSION THREAD

**Discussion by akano12** [libre-tube/LibreTube#4318] [General]
As the title says, there are plenty of issues that not only they can't be reproduced, but also stale as OPs  opened issues and left without providing any further info or trying and reporting what was suggested.

**Comment by Bnyro** [comment]
I think we should rather just close these issues, many of them are likely to be fixed in the meantime and it's better to keep as few irrelevant issues open as possible.

**Comment by FireMasterK** [comment]
https://drewdevault.com/2021/10/26/stalebot.html

I don't like stale bots either personally

**Reply by Bnyro** [reply]
Agree on that, some issues just take time to become fixed / worked on, and if they get closed automatically although they're still relevant, that'd be a bit weird.

**Comment by Silther** [comment]
the priority labels (and wontfix) that already exist actually say enough about it.

**Comment by akano12** [comment]
Not sure if there has been a misunderstanding because what everybody said so far isn't related what I'm referring to. 

1. Not talking about stale bots. 
2. I'm just suggesting this for better filtering, no more no less. 
3. I'm taking about issues where an issue is opened with little to no or if user didn't reply to what was suggested or asked for. 

Typically when closing such issues, they're marked as invalid/completed/can't reproduce even though they could be valid reports.

Take https://github.com/libre-tube/LibreTube/issues/4246 as example, it's a valid report, contains crash log and was closed probably (definitely) due to inactivity.

And @Silther ,what you said literally makes 0 sense. If you have a stale issue you don't close it and mark it as "won't fix", doesn't make any sense. 
And the 'priority' label is used to identity which features/bugs are to be prioritized over other features/bugs, I'm pretty sure a bug affecting the app performance would have a higher priority than working on a theme.
